modules/apps/object/object-service/src/main/java/com/liferay/object/internal/related/models/SystemObjectMtoMObjectRelatedModelsProviderImpl.java

```diff
@@ -224,6 +224,92 @@ public List<T> getUnrelatedModels(
 						return groupIdColumn.eq(groupId);
 					}
 				).and(
+					() -> {
+						String primaryKeyColumnName1 =
+							objectDefinition1.getPKObjectFieldDBColumnName();
+
+						Column<DynamicObjectRelationshipMappingTable, Long>
+							primaryKeyColumn1 =
+								(Column
+									<DynamicObjectRelationshipMappingTable,
+									 Long>)
+										 dynamicObjectRelationshipMappingTable.
+											 getColumn(primaryKeyColumnName1);
+
+						String primaryKeyColumnName2 =
+							objectDefinition.getPKObjectFieldDBColumnName();
```

Comment:
I'm a bit confused about the usage of the variables.
For `primaryKeyColumnName1` was used `objectDefinition1`.
For `primaryKeyColumnName2` was used `objectDefinition`.
Would not b better to use name `objectDefinition` as `objectDefinition2`?